### PR TITLE
classic-ui/view and layers

### PR DESCRIPTION
**Pull request by MrTango**

### CODE REVIEW

**Comment by stevepiercy** [comment]
@MrTango before getting too far, please make sure that you bring this PR up to date with `6-dev`, either with a merge or rebase.

**Comment by MrTango** [comment]
It`s far from perfect, but we have to start somewhere.
We should merge this after review and continue to improve it and also fill in some missing peaces to fix the references to other pages.

File: docs/classic-ui/layers.md
Comment (stevepiercy) [inline docs/classic-ui/layers.md]: The last sentence does not connect with the first. Why is it necessary to remember this? And where was the concept introduced that the reader should remember it?

File: docs/classic-ui/layers.md
Comment (stevepiercy) [inline docs/classic-ui/layers.md]: This link is a 404. I don't know what is a valid URL for this.

File: docs/classic-ui/layers.md
Comment (stevepiercy) [inline docs/classic-ui/layers.md]: How do I use this?

**Comment by ksuess** [comment]
> fill in some missing peaces

I love your typo!

File: docs/classic-ui/views.md
Comment (stevepiercy) [inline docs/classic-ui/views.md]: This reference generates a Sphinx warning and must be fixed before merging. The target link to https://docs.plone.org/develop/addons/components/genericsetup does not make sense to me, otherwise I would replace it with that. Can you suggest a better target?

File: docs/classic-ui/views.md
Comment (stevepiercy) [inline docs/classic-ui/views.md]: This reference must be fixed before merge. The target goes to the page, not any specific section in the page. https://docs.plone.org/adapt-and-extend/theming/templates_css/template_basics

File: docs/classic-ui/views.md
Comment (stevepiercy) [inline docs/classic-ui/views.md]: What is this? Please elaborate.

File: docs/classic-ui/views.md
Comment (stevepiercy) [inline docs/classic-ui/views.md]: Fix this link before merging. I don't know what would be a good target, else I would do it myself.

File: docs/classic-ui/views.md
Comment (stevepiercy) [inline docs/classic-ui/views.md]: "Zope 2"? Aren't we using Zope 5 now?

File: docs/classic-ui/views.md
Comment (stevepiercy) [inline docs/classic-ui/views.md]: "Plone 2.x"? Is this note even relevant for Plone 6? Can this note be deleted?

File: docs/classic-ui/views.md
Comment (stevepiercy) [inline docs/classic-ui/views.md]: Are we still using markers for Unicode strings in Python 3? Can this snippet be updated for Python 3?

File: docs/classic-ui/views.md
Comment (stevepiercy) [inline docs/classic-ui/views.md]: Is this true in Plone 6? Zope 2 and 3?

File: docs/classic-ui/views.md
Comment (stevepiercy) [inline docs/classic-ui/views.md]: Zope 3 again. Still needed in Plone 6?

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Whew! I pushed up a lot of changes, mostly clean up of English and MyST syntax and grammar, but there are still some questions.

I also am absolutely not in favor of merging anything with a broken link. I put in temporary fixes and added a TODO for most, but there remain a few that I have no clue as to what the original intended target should be.

Finally I asked some questions in my comments about some content that appears to be ancient (Plone 2 and Zope 2, RLY?). Was this content copy-pasted without careful review and testing? We should always review anything that gets into a PR before merging it into the docs. No cruft allowed!

**Comment by stevepiercy** [comment]
@MrTango I would appreciate a review and responses to my questions. Thank you!

**Comment by stevepiercy** [comment]
Netlify was temporarily unable to reach GitHub. See preview links:

- https://deploy-preview-1291--6-dev-docs-plone-org.netlify.app/classic-ui/views.html
- https://deploy-preview-1291--6-dev-docs-plone-org.netlify.app/classic-ui/layers.html

File: docs/classic-ui/views.md
Comment (MrTango) [reply docs/classic-ui/views.md]: In general the info still makes sense. Zope 3 is completely different to Zope2 and now Zope4.x and yes i know :laughing:. 
The five package integrates Zope 3 technologies into Zope2/Zope4 or let's just say Zope from now on.
In other places where i found Zope 2 or Zope 3 i removed the 2 or 3, but it depends on the context.
Here i would keep it.

File: docs/classic-ui/views.md
Comment (MrTango) [reply docs/classic-ui/views.md]: Since we marcos are still part of the stack and being used in core, the statement should stay.
But it needs rephrasing.

**Comment by MrTango** [comment]
@stevepiercy i hope it is much better now, we should merge this soon and improve things later on. I'll spend some time during the sprint in Innsbruck in Feb on docs. There is still a lot to do.

**Comment by jensens** [comment]
Can we get rid of all "prerendered for Plone version 4.x or greater."?

**Comment by stevepiercy** [comment]
Please give me a few days to review.

File: docs/classic-ui/layers.md
Comment (stevepiercy) [reply docs/classic-ui/layers.md]: See #1426

**Review by stevepiercy — DISMISSED** [review]
I completed my review and pushed commits.

I fixed a few broken links and syntax errors elsewhere while I was at it.

When CI is green, I will merge this PR. Thank you!

**Comment by stevepiercy** [comment]
All green. Thank you, @MrTango! What a huge effort!

**Pull request merged**
